CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 2.10.0
+
+## Features ✨ and improvements 🏁
+* Adding new marker styling option: rotationAlignment: 'horizon' allowing marker rotation to match the curvature of the horizon in globe view ([#11894](https://github.com/mapbox/mapbox-gl-js/pull/11894))
+* Improve rendering performance on globe view ([#12050](https://github.com/mapbox/mapbox-gl-js/pull/12050))
+* Improve tile loading performance on low zoom levels ([#12061](https://github.com/mapbox/mapbox-gl-js/pull/12061))
+* Improve globe-mercator transition and map load performance with globe projection ([#12039](https://github.com/mapbox/mapbox-gl-js/pull/12039))
+* Improve panning precision on Globe View and relax constrains on lower zoom levels ([#12114](https://github.com/mapbox/mapbox-gl-js/pull/12114))
+
+
+## Bug fixes 🐞
+* Fix potential flickering on on image source updates ([#11928](https://github.com/mapbox/mapbox-gl-js/pull/11928))
+* Copy stylesheet to allow toggling different styles using setStyle without overwriting some of the properties ([#11942](https://github.com/mapbox/mapbox-gl-js/pull/11942))
```

Comment:
As mentioned #11942 was introduced in 2.9.0 release. We'll have to check out the github release tools to see why it let some of these pass through!

Reply:
My guess is that GH release tools doesn't account for cherry-picking. But that would be great feature to add!

```suggestion
```